chasm/lib/activity/config.go

```diff
@@ -4,17 +4,24 @@ import (
 	"time"
 
 	"go.temporal.io/server/common/dynamicconfig"
+	"go.temporal.io/server/common/retrypolicy"
 )
 
 var (
+	Enabled = dynamicconfig.NewNamespaceBoolSetting(
+		"activity.enabled",
```

Comment:
This will be more meaningful to operators and the tense matches the rest of our configs (enable vs. enabled).

```suggestion
		"activity.enableStandalone",
```

Reply:
Changed